internal/controller/ocivalidator_controller.go

```diff
@@ -53,18 +55,14 @@ func (r *OciValidatorReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 	validator := &v1alpha1.OciValidator{}
 	if err := r.Get(ctx, req.NamespacedName, validator); err != nil {
-		// Ignore not-found errors, since they can't be fixed by an immediate requeue
-		if apierrs.IsNotFound(err) {
-			return ctrl.Result{}, nil
-		}
```

Comment:
saw this in the reference PR you shared @TylerGillson, but do we actually want to fully remove this? because we're ignoring not found errors, imo it might be better if we do something like this:

```
		if !apierrs.IsNotFound(err) {
			r.Log.Error(err, "failed to fetch OciValidator")
		}
```

without this, the plugin will always log errors whenever we delete a CR (even if thats a normal use case)